WordPress/gutenberg
# Add foundations:elevation to storybook

<img width="1588" alt="Screenshot 2024-10-15 at 14 48 15" src="https://github.com/user-attachments/assets/e05619b3-cde7-4998-b886-a6e46dcb9d47">


Related to https://github.com/WordPress/gutenberg/pull/66122.

## What?
Adds a page detailing elevation as a foundation of the overall design language. 

## Why?
To help guide end users about how to use elevation when working with the WordPress components package, or when creating new UIs.

## Testing Instructions
1. `npm run storybook:dev`
2. Observe the new Foundations / Design language / Elevation section

## Missing pieces
This PR is a draft for now. I'd welcome assistance with adding:

1. Images – it's unclear if a Storybook addon is required for this, and where any images should be hosted.
2. Internal links – I tried adding the [links addon](https://storybook.js.org/addons/@storybook/addon-links) to create an internal link from the tokens section to the tokens page, but couldn't get it working.
